### ISSUE TITLE
Suggestion to drop 3.7 in next release/ current 2022 ?

### ISSUE THREAD

**Issue by Carreau** [ipython/ipykernel#836]
That would be in line with https://numpy.org/neps/nep-0029-deprecation_policy.html, and 3.7 is the things that poses problem with importlib-metadata and co.

IPython 8 is also 3.8+

Thoughts ?

**Comment by blink1073** [comment]
Yeah, I think it makes sense for us to track what IPython is doing.

**Comment by Carreau** [comment]
Ok, once IPython 8.0 is out, and dust has settled a bit I can send a pull-request that bump minimal python versions. 

Thanks !
